internal/provider/destination_resource.go

```diff
@@ -40,15 +40,16 @@ type DestinationResource[T DestinationModel] struct {
 	toModelFunc       destinationToModelFunc[T]
 	getIdFunc         idGetterFunc[T]
 	getPipelineIdFunc idGetterFunc[T]
-	getSchemaFunc     getSchemaFunc
+	//getSchemaFunc     getSchemaFunc
```

Comment:
nit remove?

Reply:
Yup, fixed it locally and then forgot to commit. Should be pushed now.